BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -254,6 +261,22 @@ public class AppProcessor {
             services.errorReporter.log(error: error)
         }
     }
+
+    /// Starts timer to send organization events regularly
+    private func startEventTimer() {
+        sendEventTimer = Timer.scheduledTimer(withTimeInterval: 5, repeats: true) { _ in
```

Comment:
❓ This will fire every five seconds - should it be five minutes?

Reply:
🤦🏼‍♀️ Yep